src/shared/components/complex/gokb-platform-section/gokb-platform-section.vue

```diff
@@ -164,7 +164,7 @@
       showAddPlatformPopup () {
         this.addPlatformPopupVisible = true
       },
-      addNewPlatform (value) {
+      editPlatform (value) {
         this.localValue.push({ name: value.name, primaryUrl: value.primaryUrl, id: this.tempId(), isDeletable: true, unsaved: true })
         this.$emit('update', 'platforms')
       }
```

Comment:
This does not really fit with the rest of the workflow. Currently, platforms are only transfered to the backend as part of the payload when the Org itself is saved from `edit-provider-view`. Therefore new values added to the list here were marked with `unsaved: true` and given a temporary ID for selection purposes. If they are instead to be transmitted directly when hitting confirm in `gokb-edit-platform-popup`, this would not be correct behaviour. Ignoring the fact that without changes in the backend controller these would be simply saved again when the provider changes in `edit-provider-view` are submitted, the `value` here should be the (re-mapped like the other list entries) object returned from `platformServices.createOrUpdatePlatform`.

Reply:
Waiting for the response of the users about preferred behaviour.

Reply:
Changed as discussed.